conf/app.routes

```diff
@@ -38,3 +38,13 @@ GET        /trader/:ern/draft/:lrn/how-movement-transported
 POST       /trader/:ern/draft/:lrn/how-movement-transported                       controllers.sections.journeyType.HowMovementTransportedController.onSubmit(ern: String, lrn: String, mode: Mode = NormalMode)
 GET        /trader/:ern/draft/:lrn/how-movement-transported/change                controllers.sections.journeyType.HowMovementTransportedController.onPageLoad(ern: String, lrn: String, mode: Mode = CheckMode)
 POST       /trader/:ern/draft/:lrn/how-movement-transported/change                controllers.sections.journeyType.HowMovementTransportedController.onSubmit(ern: String, lrn: String, mode: Mode = CheckMode)
+
+GET        /trader/:ern/draft/:lrn/how-movement-transported                       controllers.sections.journeyType.HowMovementTransportedController.onPageLoad(ern: String, lrn: String, mode: Mode = NormalMode)
+POST       /trader/:ern/draft/:lrn/how-movement-transported                       controllers.sections.journeyType.HowMovementTransportedController.onSubmit(ern: String, lrn: String, mode: Mode = NormalMode)
+GET        /trader/:ern/draft/:lrn/how-movement-transported/change                controllers.sections.journeyType.HowMovementTransportedController.onPageLoad(ern: String, lrn: String, mode: Mode = CheckMode)
+POST       /trader/:ern/draft/:lrn/how-movement-transported/change                controllers.sections.journeyType.HowMovementTransportedController.onSubmit(ern: String, lrn: String, mode: Mode = CheckMode)
```

Comment:
These are a duplicate of the routes above, probably slipped through while fixing a merge conflict

Reply:
Pushed commit to fix https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/22/commits/950fb6cd65eabe6849d13c87eef73be93b48c602